docs/versioned_docs/version-2.11.0/stateful-applications/cockroachdb.md

```diff
@@ -0,0 +1,294 @@
+---
+id: cockroachdb
+title: OpenEBS for CockroachDB
+---
+
+![OpenEBS and CockroachDB](../assets/o-cockroachdb.png)
+## Introduction
+
+CockroachDB is a cloud-native SQL database for building global, scalable cloud services that survive disasters. It is a distributed SQL database built on a transactional and strongly consistent key-value store. It scales horizontally; survives disk, machine, rack, and even data center failures with minimal latency disruption and no manual intervention; supports strongly consistent ACID transactions; and provides a familiar SQL API for structuring, manipulating, and querying data.
+
+This guide explains the basic installation for CockroachDB operators on OpenEBS Local PV devices.
+## Deployment model 
+
+[![OpenEBS and CockroachDB](../assets/Local-PV-Distributed-device-cockroachdb.svg)](../assets/Local-PV-Distributed-device-cockroachdb.svg)
+
+The Local PV volume will be provisioned on a node where CockroachDB pods are getting scheduled and uses one of the matching unclaimed block devices for each of them, which will then use the entire block device for storing data. No other application can use this device. If users have limited block devices attached to some nodes, they can use `nodeSelector` in the application YAML to provision applications on particular nodes where the available block device is present. The recommended configuration is to have at least three nodes and one unclaimed external disk to be attached per node. 
+## Configuration workflow 
+
+1. [Install OpenEBS](#install-openebs)
+2. [Select OpenEBS storage engine](#select-openebs-storage-engine) 
+3. [Configure OpenEBS Local PV StorageClass](#configure-openebs-local-pv-storageclass) 
+4. [Install CockroachDB Operator](#install-cockroachdb-operator)
+5. [Accessing CockroachDB](#accessing-cockroachdb)
+### Install OpenEBS
```

Comment:
```
### Install OpenEBS {#install-openebs}
```

Reply:
Its not needed, the id will be generated automatically.
The generated id will be in lowercase hyphen separated no special chars

Reply:
ok